docker/Dockerfile

```diff
@@ -40,13 +40,6 @@ RUN \
   npm install --global remark-cli remark-lint-no-dead-urls remark-validate-links; \
   remark --help
 
-# TODO (#825): remove this `RUN`
```

Comment:
Pls also change image version in .circleci/config.yml to 8.0.0 (I've pushed it already)